alacritty_terminal/src/ansi.rs

```diff
@@ -1442,39 +1444,56 @@ mod tests {
     };
     use crate::index::{Column, Line};
     use crate::term::color::Rgb;
-    use std::io;
+    use std::io::{self, sink};
 
-    /// The /dev/null of `io::Write`
-    struct Void;
+    struct PseudoTermHandler {
```

Comment:
I think the word `Term` is a bit misleading here, since it doesn't handle terminals, but it handles escapes. So maybe something more like `PseudoEscapeHandler` would fit better. Though we don't need to be overly specific either, so we can also just do `PseudoHandler`.

I'm also wondering if we should use a name more commonly understood as a test replacement method. So maybe `MockHandler` or something like that would express the intention better? Especially because a `pseudoterminal` is something that does exist (PTY) and I don't think it's accurate here.